.circleci/workflows.yml

```diff
@@ -500,7 +500,7 @@ jobs:
       psql_version:
         type: string
     machine:
-      image: ubuntu-2204:2022.07.1
+      image: ubuntu-2204:2024.04.4
       docker_layer_caching: true
```

Comment:
tests are failing
    TypeError: Cannot assign to read only property 'fetch' of object '[object global]'

      28 | import cheerio from 'cheerio';
      29 |
    > 30 | global.fetch = fetch;

Reply:
I push another commit to upgrade Jest.
Looks like the tests are running now; Waiting the end of execution to confirm.

Reply:
all tests ok :heavy_check_mark: